From 2c5ad267a7eddcc757eceda817098612c805cbdd Mon Sep 17 00:00:00 2001
From: Julien Puydt <julien.puydt@laposte.net>
Date: Tue, 6 Oct 2015 08:25:56 +0200
Subject: use setuptools for everything (so python*:Depends works)

Forwarded: no
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- traitlets.orig/setup.py
+++ traitlets/setup.py
@@ -98,8 +98,7 @@
     ],
 )
 
-if 'develop' in sys.argv or any(a.startswith('bdist') for a in sys.argv) or any(a.startswith('sdist') for a in sys.argv):
-    import setuptools
+import setuptools
 
 setuptools_args = {}
 
