Description: allow reproducible builds in projects using traitlets
Author: Chris Lamb
Forwarded: https://github.com/ipython/traitlets/pull/535

--- a/traitlets/traitlets.py
+++ b/traitlets/traitlets.py
@@ -2554,6 +2554,9 @@
     def subclass_init(self, cls: type[t.Any]) -> None:
         pass  # fully opt out of instance_init
 
+    def make_dynamic_default(self):
+        # Ensure value is sorted for a reproducible build
+        return sorted(super(Set, self).make_dynamic_default())
 
 def _validate_bounds(
     trait: Int[t.Any, t.Any] | Float[t.Any, t.Any], obj: t.Any, value: t.Any
