Description: allow reproducible builds in projects using traitlets
Author: Chris Lamb
Forwarded: https://github.com/ipython/traitlets/pull/535

--- traitlets.orig/traitlets/traitlets.py
+++ traitlets/traitlets/traitlets.py
@@ -2357,6 +2357,9 @@
     def info_rst(self):
         return self._info(as_rst=True)
 
+    def make_dynamic_default(self):
+        # Ensure value is sorted for a reproducible build
+        return sorted(super(Set, self).make_dynamic_default())
 
 class FuzzyEnum(Enum):
     """An case-ignoring enum matching choices by unique prefixes/substrings."""
